tools/provision/formula/augeas.rb

```diff
@@ -4,15 +4,15 @@ class Augeas < AbstractOsqueryFormula
   desc "A configuration editing tool and API"
   homepage "http://augeas.net/"
   url "https://github.com/hercules-team/augeas.git",
-    :revision => "aacc8ac07f6722622b73d9183b9acc666906d2e9"
-  version "1.8.1"
+    :revision => "3775c2bf53fef5f694fcf25308cee1dfe00600c4"
+  version "1.9.0"
   revision 100
 
   bottle do
     root_url "https://osquery-packages.s3.amazonaws.com/bottles"
     cellar :any_skip_relocation
     sha256 "3e09566342fbb532676db198d8c3b5c5bacdb3bc633dd751f19ac2a3b184631c" => :sierra
```

Comment:
Can you update the `sierra` hash too: 4d64984c408db64f33e4ba629195caa96ede0e02c1bcb7154fc2c9dc195ff4e9

Reply:
Done :)